test/2_create_test_app_namespace.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 announce "Creating Test App namespace."
 
 if [[ $PLATFORM == openshift ]]; then
-  oc login -u $OPENSHIFT_USERNAME
+  $cli_with_timeout "login -u $OPENSHIFT_USERNAME"
```

Comment:
```suggestion
  $cli_with_retry "login -u $OPENSHIFT_USERNAME"
```

Reply:
i'm no sure retry is the right name here since we give timeout as parameter and not the retry number it may confuse the user.